Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1beta3/defaults.go

```diff
@@ -237,4 +238,46 @@ func defaultSecurityContextConstraints(scc *SecurityContextConstraints) {
 		scc.AllowEmptyDirVolumePlugin = new(bool)
 		*scc.AllowEmptyDirVolumePlugin = true
 	}
+
+	defaultSecurityContextConstraintsVolumes(scc)
+}
+
+// defaultSecurityContextConstraintsVolumes defaults the volume slice of the SCC.
+// In order to support old clients the boolean fields will always take precedence.
+func defaultSecurityContextConstraintsVolumes(scc *SecurityContextConstraints) {
```

Comment:
this depends on being called after other defaulting bits... don't split it into a function that could be called incorrectly, just keep these bits in the defaultSecurityContextConstraints func